frontend/src/Pages/NewKeepitPage.js

```diff
@@ -0,0 +1,52 @@
+import { useHistory } from 'react-router-dom'
+import { useState, useEffect } from 'react'
+import UploadButton from '../Components/UploadButton'
+
+export default function NewKeepitPage() {
+  const history = useHistory()
+  const [images, setImages] = useState()
+  const [showUploadButton, setShowUploadButton] = useState(false)
```

Comment:
You could remove this state and effect if you do this:

```suggestion
  const showUploadButton = images && images.length > 0;
```

or like in the next comment suggested:

```suggestion
  const showUploadButton = Boolean(images?.length);
```

Reply:
Great, thanks!